### PR TITLE
Change homepage title

### PR DESCRIPTION
**Pull request by rudygasson**
Please check out, nice new title! 
Screenshot 2024-02-03 at 11 22 24

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h1>Welcome</h1>
+<p>This is my app</p>
```
Comment (acambiaso93) [inline app/views/pages/home.html.erb:2]: like it!

**Review by acambiaso93 — COMMENTED** [review]
look!

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h1>Welcome</h1>
```
Comment (acambiaso93) [inline app/views/pages/home.html.erb:1]: yes

**Review by acambiaso93 — APPROVED** [review]

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (rudygasson) [reply app/views/pages/home.html.erb:1]: Thank you!

**Pull request merged**